backend/src/main/java/org/eclipse/tractusx/semantics/hub/persistence/triplestore/SparqlQueries.java

```diff
@@ -226,16 +222,9 @@ public static Query echoQuery() {
     * @param pageSize the page size
     * @return a Sparql query with the provided search filters
     */
-   public static Query buildFindAllQuery( String namespaceFilter, String nameFilter, String nameType, ModelPackageStatus status,
+   public static Query buildFindAllQuery( String namespaceFilter, ModelPackageStatus status,
          int page, int pageSize ) {
 
-      if(StringUtils.isNotBlank(nameType) && StringUtils.isNotBlank(nameFilter)){
-         final ParameterizedSparqlString pss = buildExtendedSearchQuery( FIND_ALL_EXTENDED_QUERY, namespaceFilter,
```

Comment:
It seems like you could remove the static string defining the FIND_ALL_EXTENDED_QUERY since it is not used somewhere else.